tests/unit/test_all_paths.cpp

```diff
@@ -307,3 +308,33 @@ TEST_F(AllPathsTest, DestinationSpecificPaths) {
 	Graph_Free(g);
 }
 
+
+// Test all paths of the minimum length from source to a specific destination node.
+TEST_F(AllPathsTest, OnlyShortestPaths) {
```

Comment:
Please reconsider, in-case this is covered by the python flow test, simply remove.